app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,23 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
+      if count_only
+        grouped_rates.length
+      else
+        objects = []
+        grouped_rates.sort.each do |description_group|
+          objects.push(
+            :id    => description_group[0],
+            :text  => description_group[0],
```

Comment:
I had used the rate description like identifier. This generates errors when the description of a rate is edited. Thanks for the correction, without it,  we would not have realized  the problem.